cockroachcloud/cmek-faq.md

```diff
@@ -19,7 +19,13 @@ Yes, the data encryption key is rotated automatically once every month. It’s n
 
 ## Can we rotate the CMEK for a cluster after a certain time or at some periodic interval?
 
-Not yet. The ability to rotate CMEK would be available at a later time. Once that’s ready, you could do it ad-hoc, or at a periodic interval using your own managed scheduling mechanism.
+Yes. Two different types of rotation are possible:
+
+- When you rotate a CMEK key within your KMS platform, a new version of the key is created with new key material. In both GCP KMS and AWS KMS, encryption operations automatically uses the active key version, while decryption operations automatically use the key version that was used to encrypt the data. For this reason, {{ site.data.products.dedicated }} does not need any awareness of rotation operations within your KMS platform.
```

Comment:
I verified that a customer can force CC to start using the new version via a rotate ([slack](https://cockroachlabs.slack.com/archives/C02Q8MLFFE0/p1657210417321139?thread_ts=1657120148.107949&cid=C02Q8MLFFE0)). You think thats worth mentioning? If the customer never rotates, we'll never encrypt with the latest version.

Reply:
Wow that's a pretty significant caveat. Yes, we should definitely mention it.